meinberlin/assets/scss/components/_list_item.scss

```diff
@@ -130,20 +137,3 @@
 .list-item__status--active {
     color: $primary;
 }
-
-.list-item__header {
-    display: flex;
-    flex-direction: column-reverse;
-}
-
-@media screen and (min-width: $breakpoint-xs) {
-    .list-item__vote {
-        display: flex;
-        justify-content: space-between;
-        align-items: center;
-    }
-
-    .list-item__header {
-        flex-direction: row;
-    }
-}
```

Comment:
@goapunk this commit for some reason reset the other changes from the first commit, no clue why?!